hack/buildconfigs-template.json

```diff
@@ -0,0 +1,329 @@
+{
+   "apiVersion":"v1",
+   "objects":[
+      {
+         "apiVersion":"v1",
+         "kind":"ImageStream",
+         "metadata":{
+            "name":"java-jdk-8"
+         },
+         "spec":{
+            "dockerImageRepository":"jboss/base-jdk:8"
+         }
+      },
+	{
+         "apiVersion":"v1",
+         "kind":"ImageStream",
+         "metadata":{
+            "name":"eap-openshift"
+         },
+         "spec":{
+            "dockerImageRepository":"registry.access.redhat.com/jboss-eap-6/eap-openshift"
+         }
+      },
+	{
+         "apiVersion":"v1",
+         "kind":"ImageStream",
+         "metadata":{
+            "name":"heapster"
+         },
+         "spec":{
+            "dockerImageRepository":"burmanm/hawkular-heapster:0.18.3"
```

Comment:
This needs to be changed since we are now using openshift/origin-metrics-heapster-base:latest